Dangerfile

```diff
@@ -28,8 +28,8 @@ end
 
 labels_checker.check(
   do_not_merge_labels: ['do not merge'],
-  required_labels: [//],
-  required_labels_error: 'PR requires at least one label.'
+  required_labels: [/^\[Type\]/, /^(\[Area\]|\[Project\])/],
+  required_labels_error: 'PR requires a [Type] label and either a [Area] or [Project] label.',
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-android/blob/8b9a246c3ceb19d3086c6c5e5639b5569e2276d2/Dangerfile#L32"></span>Style/TrailingCommaInArguments: Avoid comma after the last parameter of a method call.</td>
    </tr>
  </tbody>
</table>